.github/workflows/release.yml

```diff
@@ -35,6 +35,7 @@ jobs:
         uses: python-semantic-release/python-semantic-release@v8.0.4
         with:
           github_token: ${{ secrets.RELEASE }}
+          changelog: "false"
```

Comment:
@MartinBernstorff you might find this line interesting

Reply:
@imenelydiaker it will still be added to the releases and since we do not require everyone to use the prefixed (e.g. "docs:"), the changelog would be very messy.